extension/android/CMakeLists.txt

```diff
@@ -129,6 +142,12 @@ if(EXECUTORCH_BUILD_LLAMA_JNI)
     quantized_kernels
     quantized_ops_lib
   )
+  ADD_LIBRARY(libneuron_buffer_allocator SHARED IMPORTED)
+  FIND_LIBRARY(libneuron_buffer_allocator_LIBRARY libneuron_buffer_allocator PATHS /home/hsz/e3/executorch/libneuron_buffer_allocator.so)
```

Comment:
Have problem with this part

Reply:
Won't be able to land this as-is I believe, given the home dir, etc

Reply:
It's a draft. It's not working, even with local stuff

Reply:
I've tried removing " FIND_LIBRARY(libneuron_buffer_allocator_LIBRARY libneuron_buffer_allocator PATHS /home/hsz/e3/executorch/libneuron_buffer_allocator.so)"

and set the path at
  SET_PROPERTY(TARGET libneuron_buffer_allocator PROPERTY IMPORTED_LOCATION path/to/libneuron_buffer_allocator.so)

Then it worked on my side

reference:
https://stackoverflow.com/a/62153332

Reply:
thanks @neuropilot-captain . I'm trying this out now. 

Do you know if/how we need to account for libneuron_backend.so as well or that is not necessary?

Reply:
if I remove the FIND_LIBRARY command and replace it with the SET_PROPERTY command you mentioned then I get the following error:
```
CMake Error in CMakeLists.txt:
  IMPORTED_LOCATION or IMPORTED_IMPLIB not set for imported target
  "libneuron_buffer_allocator" configuration "Release".
```

I suspect this is because of the SET_PROPERTY in line 147 and 148. On removing those and trying to build, I get the error:
```
ld: error: /Users/cmodi/Documents/ai/clean/executorch/backends/mediatek/libneuron_buffer_allocator.so is incompatible with elf_x86_64
```

@neuropilot-captain any thoughts on this issue?